coord/src/rmr/test/test_io_runtime_ctx.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#include "minunit.h"
+#include "coord/src/rmr/rq.h"
+#include "info/global_stats.h"
+#include "rmutil/alloc.h"
+#include <unistd.h>
+#include <stdatomic.h>
+#include <uv.h>
+
+uv_thread_t loop_th;
+static atomic_bool loop_started = false;
+
+/* start the event loop side thread */
+static void sideThread(void *arg) {
+  atomic_store(&loop_started, true);  // Signal that loop thread is starting
+  while (1) {
```

Comment:
why do u have this loop? uv_run should be enough it actually runs the event loop